### PR TITLE
Add 'Browse all' header button

### PR DESCRIPTION
**Pull request by Sinclert**
This PR addresses issue https://github.com/nyudlts/ultraviolet/issues/56, by adding a new template override (`header.html`), extending it to add a new _"Browse all"_ button within the horizontal menu. The functionality of the button is exactly the same as clicking the 🔍  button with no parameters, it has been added for accessibility purposes.

**Minor**: The dependencies within [Pipfile](https://github.com/nyudlts/ultraviolet/blob/main/Pipfile) have been arranged alphabetically to calm my TOC.

### CODE REVIEW

**Comment by andrewbattista** [comment]
Looks great to me!

**Review by ekate — APPROVED** [review]
Looks good. Will merge after @mnyrop approves

**Review by mnyrop — APPROVED** [review]

**Pull request merged**
